tests/spec/puppeteer-axe-spec.js

```diff
@@ -0,0 +1,37 @@
+/* Puppeteer tests for the A11y. */
+const { AxePuppeteer } = require('axe-puppeteer');
+const puppeteer = require('puppeteer');
+
+describe("Puppeteer accessibility tests for the Python Editor.", function() {
+    'use strict';
+
+    const editorURL = 'http://localhost:5000/editor.html';
+    let browser = null;
+
+    beforeAll(async () => {
+        // Setup a headless Chromium browser.
+        // Flags allow Puppeteer to run within a container.
+        browser = await puppeteer.launch({
+            headless: true,
+            args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-dev-shm-usage"]
+        });
+        const page = await browser.newPage();
+ 
+        await page.goto(editorURL);    
+ 
+        await page.close();
+        });
+
+    afterAll(async () => {
+    browser.close();
+    });
+
+    test( 'Checks the editor.html page with Axe', async () => {
+        // First, run some code which loads the content of the page.
+        const page = await browser.newPage();
+        await new AxePuppeteer(page).analyze();
+        await expect(page).toPassAxeTests({
+            disabledRules: [ 'document-title', 'page-has-heading-one', 'html-has-lang', 'landmark-one-main' ],
```

Comment:
`document-title`:
> Ensures each HTML document contains a non-empty <title> element

Why is this needed? Our title is populated.

Reply:
I think this is because the title is provided by the versioner. If I run the test with `document-title` enabled against python.microbit.org it passes. If I run it against localhost:5000/editor.html it fails

Reply:
But these tests are running here for the editor without the versioner, so we should remove the exception and it should pass without it, right?

Reply:
The issue was not running the test against the editor properly I wasn't navigating to the editorURL just running an instance of Chromium :)